suites/message/message_application.go

```diff
@@ -46,13 +46,26 @@ func MessageTest_MessageApplicationEdgecases(t *testing.T, factory state.Factori
 			td.MessageProducer.Transfer(alice, alice, chain.Value(transferAmnt), chain.Nonce(0), chain.GasPrice(10), chain.GasLimit(1)),
 			exitcode.SysErrOutOfGas)
 
-		// Expect Message application to fail due to lack of gas when sender address is unknown
+		// Expect Message application to fail due to lack of gas when the message being sent has invalid destination.
 		unknown := utils.NewIDAddr(t, 10000000)
 		td.ApplyFailure(
 			td.MessageProducer.Transfer(alice, unknown, chain.Value(transferAmnt), chain.Nonce(0), chain.GasPrice(10), chain.GasLimit(1)),
 			exitcode.SysErrOutOfGas)
 	})
 
+	t.Run("fail not enough gas to cover account actor creation", func(t *testing.T) {
+		td := builder.Build(t)
+		defer td.Complete()
+
+		alice, _ := td.NewAccountActor(drivers.SECP, aliceBal)
+		newAccount := utils.NewSECP256K1Addr(t, "1")
+
+		// Expect Message application to fail due to lack of gas
+		td.ApplyFailure(
+			td.MessageProducer.Transfer(newAccount, alice, chain.Value(transferAmnt), chain.Nonce(0), chain.GasPrice(1), chain.GasLimit(200)),
```

Comment:
I think we can do better. How about:
- Start by finding the true gas cost by sending a successful message.
- Then try every gas value mod `something` less than the true cost, and check we get the same exit code

`something` could probably be as large as 50 and still very robust to the gas table changing (since the true cost will be >500).